CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 ### Changed
 - When editing an article, the tab "Optional fields" now shows "ISSN"
 - When editing a book, the tab "Optional fields" now shows "ISBN"
+- When using "Copy citation (HTML)" and pasting into Notepad++, plain text is always pasted.
```

Comment:
I doubt it only affects Notepad++, so I would opt for describing it more general, e.g. text Editor